tools/experiments/experiments.js

```diff
@@ -338,6 +338,16 @@ const EXPERIMENTS = [
     name: 'Enables scroll snap on carousel across all browsers/OSes',
     cleanupIssue: 'https://github.com/ampproject/amphtml/issues/16508',
   },
+  {
+    id: 'amp-carousel-new-arrows',
```

Comment:
`amp-carousel-new-arrows` looks like a bad merge issue.

Reply:
Yup it was, reverted.